pybleau/app/model/dataframe_plot_manager.py

```diff
@@ -543,11 +600,40 @@ def action_edit_style_requested(self, manager, attr_name, new):
     @on_trait_change("contained_plots:plot_factory:context_menu_manager:"
                      "delete_requested", post_init=True)
     def action_delete_requested(self, manager, attr_name, new):
-        """ A factory requested its style to be edited. Launch dialog.
+        """ A factory requested a plot be deleted. Launch dialog.
         """
         desc = self._get_desc_for_menu_manager(manager)
         desc.container_idx = CONTAINER_IDX_REMOVAL
 
+    @on_trait_change("contained_plots:plot_factory:context_menu_manager:"
+                     "template_requested", post_init=True)
+    def action_template_requested(self, manager, attr_name, new):
+        """ A factory requested a plot template be created. Launch dialog.
+        """
+        interactor = self.template_interactor
+        if interactor is None:
+            return
+
+        desc = self._get_desc_for_menu_manager(manager)
+        if desc is None:
+            return
+
+        template_name = self._request_template_name_with_desc(desc)
+        if template_name is None:
+            return
+
+        filepath = os.path.join(interactor.get_template_dir(), template_name +
+                                interactor.get_template_ext())
+        saver = interactor.get_template_saver()
+        try:
+            saver(filepath, desc.plot_config)
+        except ValueError:
+            msg = f"The {type(interactor)}'s save function requires a " \
+                  f"filepath and a plot configurator to properly save a " \
```

Comment:
Suggestion: `requires...` -> `...is expected to receive a filepath and a Configurator object to save to a template file.` ?